tests/driver_at86rf2xx/ng_at86rf2xx_params.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   tests_at86rf2xx
+ * @brief     generic at86rf231 pin config
+ * @{
+ * @file
+ * @author Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef NG_AT86RF2XX_PARAMS_H
+#define NG_AT86RF2XX_PARAMS_H
+
+#warning Using default AT86RF2XX parameters.
+
+/**
+ * @name AT86RF231 configuration
+ */
+static const  at86rf2xx_params_t at86rf2xx_params[] =
+    {
+        {
+            .spi = SPI_0,
+            .spi_speed = SPI_SPEED_1MHZ,
+            .cs_pin = GPIO_0,
+            .int_pin = GPIO_1,
+            .sleep_pin = GPIO_2,
+            .reset_pin = GPIO_3,
```

Comment:
here you break the test for the iot-lab and the samr21! The use different configurations for the device. Also this must be settable in the Makefile, as this driver should also work for boards that do not have the radio on-board, but where the radio is connected via jumper wires to any arbitrary setup.